siriuspy/siriuspy/sofb/main.py

```diff
@@ -942,6 +950,10 @@ def _do_auto_corr(self):
                 self._update_log(msg)
                 _log.info(msg)
 
+        if self._loop_state == self._csorb.LoopState.Closed:
+            self._loop_state = self._csorb.LoopState.Open
+            self.run_callbacks('LoopState-Sel', 0)
```

Comment:
maybe we could use `self.run_callbacks('LoopState-Sel', self._csorb.LoopState.Open)`, just to avoid hardcoded constants